osc-dm-product-srv/src/server.py

```diff
@@ -226,6 +226,60 @@ async def dataproducts_uuid_metrics_get(uuid: str):
     return response
 
 
+
+@app.post(ENDPOINT_PREFIX + "/products/generate")
+async def dataproducts_generate_product_post(product: models.ProductGen):
+
+    """
+    Get metrics information
```

Comment:
pretty sure the documentation description for this method is wrong, and that this is a copy-paste issue.

Reply:
updated documentation